src/package/components/Search/stories/examples/EgMultiSelectLimit.tsx

```diff
@@ -4,9 +4,9 @@ import { SearchOptionType, SearchValueType } from "../../types";
 
 export function EgMultiSelectLimit() {
   const defaultEntry: SearchOptionType[] = [];
-  const [selectedEntry, setSelectedEntry] = useState<SearchOptionType[]>(
-    defaultEntry
-  );
+  const [selectedEntry, setSelectedEntry] = useState<
+    readonly SearchOptionType[]
```

Comment:
@Methuselah96 I'm a little confused. What is the significance of declaring this type as `readonly`?

Reply:
It just means that you can't directly modify `selectedEntry` using array methods that modify the array like `push`, etc., you have to use `setSelectedEntry`. The reason why it's necessary is because `OptionsType` is defined as `ReadonlyArray<OptionType>` [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-select/src/types.d.ts#L8).

Reply:
Got it, thanks a lot!